recipes/LibriSpeech/ASR/seq2seq/train.py

```diff
@@ -292,7 +293,44 @@ def text_pipeline(wrd):
     sb.dataio.dataset.set_output_keys(
         datasets, ["id", "sig", "wrd", "tokens_bos", "tokens_eos", "tokens"],
     )
-    return train_data, valid_data, test_datasets
+    train_batch_sampler = None
```

Comment:
i had to add some lines to the train.py of LibriSpeech seq2seq and TIMIT seq2seq recipes. These however are not used by default

Reply:
yeah, this however, might be something to discuss at a high level. Do we wish to duplicate the recipe to avoid having "unused" code, or do we wish to have complete recipes that might appear to be harder to read ? I don't have strong preferences personally.

Reply:
right now we have lot of duplication also for the sorting part. 
Probably we could integrate sorting and dynamic batching and dataset creation from .json and .csv in the toolkit. So most recipes could avoid this boilerplate code